### ISSUE TITLE
add more decimal figures to FITS header values

### ISSUE THREAD

**Issue by EastEriq** [EranOfek/AstroPack#456]
As of today the format seems to be `%.4f`. Eran says that four is not enough, and once there were more figures.
I.e.
```
M_RA    =             311.4326 /                                                
M_DEC   =              30.0073 /                                                
EQUINOX =                2000. /                                                
M_AZ    =             359.9885 /                                                
M_ALT   =             -29.9397 /                                                
M_JRA   =             311.1752 /                                                
M_JDEC  =              29.9473 /                                                
M_ARA   =             311.4333 /                                                
M_AHA   =             179.9875 /                                                
M_ADEC  =              30.0073 /                                                
M_ADRA  =             311.4333 /                                                
M_ADHA  =             179.9875 /                                                
```

**Comment by EastEriq** [comment]
Not your bug, sorry, please ignore.
That came from my code, corrected now by https://github.com/PolishookDavid/LAST_OCS/commit/16fa4a00b850cc50d304a86d2cbff1932ff3fddc.

**Comment by EranOfek** [comment]
Why it was closed?

**Comment by EastEriq** [comment]
> Not your bug, sorry, please ignore. That came from my code, corrected now by [PolishookDavid/LAST_OCS@16fa4a0](https://github.com/PolishookDavid/LAST_OCS/commit/16fa4a00b850cc50d304a86d2cbff1932ff3fddc).

Read above?
